integration/spark/buildDependencies.sh

```diff
@@ -3,6 +3,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
 set -x -e
-(cd ../../client/java && ./gradlew clean publishToMavenLocal -x test)
+(cd ../../client/java && ./gradlew clean generateCode publishToMavenLocal -x test --rerun-tasks)
```

Comment:
why do you want to use this together with `clean` ?

Reply:
You risk a situation when you add a facet, but when using local maven repository you don't see it. The previous line worked if there was regular code change, but not when `OpenLineage.java` was changed.